arm-advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,641 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "2016-07-12-preview",
+		"title": "AdvisorManagementClient",
+		"description": "REST APIs for Azure Advisor"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"consumes": ["application/json"],
+	"produces": ["application/json"],
+	"paths": {
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/generateRecommendations": {
+			"post": {
+				"tags": ["GenerateRecommendations"],
+				"description": "Initiates the recommendation generation/computation process for a subscription. This is an asynchronous operation. The generated recommendations are stored in a cache in the Advisor service.",
+				"operationId": "Recommentations_Generate",
+				"parameters": [{
+					"$ref": "#/parameters/subscriptionIdParameter"
+				},
+				{
+					"$ref": "#/parameters/apiVersionParameter"
+				}],
+				"responses": {
+					"202": {
+						"description": "Accepted",
+						"headers": {
+							"Location": {
+								"description": "The URL where the status of the asynchronous operation can be checked.",
+								"type": "string"
+							},
+							"Retry-After": {
+								"description": "The delay to use while checking the status of the operation. The value is in seconds.",
+								"type": "string"
```

Comment:
Just make sure when you move to datetime that'd a breaking change when you do type: datetime from string.